src/gleam/string.gleam

```diff
@@ -157,25 +157,34 @@ pub fn slice(from string: String, at_index idx: Int, length len: Int) -> String
   }
 }
 
-// TODO
-// Drop *n* Graphemes from the left side of a
-//
-// ## Examples
-//    > drop_left(from: "The Lone Gunmen", up_to: 2)
-//    "e Lone Gunmen"
-//
-//
-// pub fn drop_left(from string: String, up_to num_graphemes: Int) -> String {}
-// TODO
-// Drop *n* Graphemes from the right side of a
-//
-// ## Examples
-//    > drop_right(from: "Cigarette Smoking Man", up_to: 2)
-//    "Cigarette Smoking M"
-//
-//
-// pub fn drop_right(from string: String, up_to num_graphemes: Int) -> String {}
+/// Drop *n* Graphemes from the left side of a string.
+///
+/// ## Examples
+///    > drop_left(from: "The Lone Gunmen", up_to: 2)
+///    "e Lone Gunmen"
 ///
+///
+pub fn drop_left(from string: String, up_to num_graphemes: Int) -> String {
+  case num_graphemes < 0 || num_graphemes > length(string) {
```

Comment:
This traverses the full string. Do we need the `num_graphemes > length(string)` at all? I think the slice function will handle that.

Reply:
Apparently the `slice` crashed on negative lengths, I fixed that. After that I could remove `num_graphemes > length(string)`.